### ISSUE TITLE
Add request details to readme.md

### ISSUE THREAD

**Issue by crutchcorn** [crutchcorn/pinbadge-icon-theme#11]
For now, anyone can request icons in a TON of ways in many formats. In the future, this should be streamlined and made clear on how to request icons

**Comment by danielhickman** [comment]
Sorta like [this](https://github.com/materialos/Icons/issues/new?title=[Request]%20Icon%20Name&body=-%20[%20]%20Icon%20Name) one for MaterialOS?
Or where you thinking of a way for people without GitHub's?
Is there package names or classes you need too (like Android)?

**Comment by crutchcorn** [comment]
No. I could do that myself from the get go if I wanted to. I appreciate the thought though.
Yeah - I was thinking a way for people with and without GitHubs. And a way to get the names of the icons from the desktop files

**Comment by loctauxphilippe** [comment]
@crutchcorn what do you mean for people without github? without all the markdown?
I can do that, if it's what you're thinking about :)
